tests/server/tso/allocator_test.go

```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build tso_full_test tso_function_test
```

Comment:
What is `tso_full_test` used for?

Reply:
Just in case we want to run all TSO tests by `-tags tso_full_test`.